project/repo.sls

```diff
@@ -34,8 +34,8 @@ project_repo:
        - cmd: project_repo
   {% else %}
   git.latest:
-    - name: "{{ pillar['repo']['url'] }}"
-    - rev: "{{ pillar['repo'].get('branch', 'master') }}"
+    - name: "{{ pillar['repo'] }}"
+    - rev: "{{ pillar.get('branch', 'master') }}"
```

Comment:
Could we maintain backward compatibility by using something like:

      - rev: "{{ pillar.get('branch', pillar['repo'].get('branch', 'master')) }}"